python/lsst/pipe/analysis/plotRhoStatistics.py

```diff
@@ -0,0 +1,347 @@
+import numpy as np
+from matplotlib import pyplot as plt
+import treecorr
+
+from lsst.daf.persistence import Butler
+import lsst.afw.table as afwTable
+from utils import E1, E2, E1Resids, E2Resids, E1ResidsHsmRegauss, E2ResidsHsmRegauss, PsfTraceSizeDiff
+
+def corr2(ra, dec, g1a, g2a, g1b, g2b, ra_units="degrees", dec_units="degrees", **treecorr_kwargs):
+    """ Function to compute auto-/cross-correlations between atmost two shear fields.
+
+        This is used to compute Rowe statistics, given the appropriate 'shear' fields.
```

Comment:
I think I still want to hold on to shear or shear-like fields with two components. Or at least ellipticity. Because a different routine has to be called to compute 2PCF for scalar fields or scalar and shear-like fields.

Reply:
"shear-like" sounds good!